### ISSUE TITLE
Style mismatch issue when node_modules is excluded from babel in SSR

### ISSUE THREAD

**Issue by gtolarc** [EOSIO/ual-reactjs-renderer#78]
When using withUAL HOC in SSR I get the following error:
![sample](https://user-images.githubusercontent.com/8623798/65405434-120d8200-de17-11e9-86d6-9931aa441cea.png)

In next.js, node_modules is excluded from transpiling by default, and there seems to be a mismatch of code on the server and client sides.

So, it would be nice to fix it like this:
https://github.com/EOSIO/ual-reactjs-renderer/blob/ee45df97ec3bff18384d6e584e82b18902414613/src/styles/base.js#L11
->
```
export const baseFont = '@import https://fonts.googleapis.com/css?family=Source+Sans+Pro;';
```

If it's ok, I would like to apply my suggestion to the code.
Or is there a more fundamental solution to the problem?

**Comment by GreenBusDriver** [comment]
@gtolarc That would be great.  Please do create a PR and tag me in it.
I just verified the syntax you're suggesting.  I don't see anything wrong with it, if it fixes an SSR issue.
Thanks for reporting this!  I'll watch for the PR.

**Comment by gtolarc** [comment]
@GreenBusDriver When will it be reflected in the release version? I hope it will be applied soon.

**Comment by bradlhart** [comment]
@gtolarc Your changes are now available on v0.2.0.  Thank you for contributing!
